m4_sessions/automation/session_one/six_rebekah_nickg_molly_jeff.md

```diff
@@ -1,3 +1,57 @@
 ## Group Six's Session Plan
 
 M4 Student Leaders: Rebekah, Nick G, Molly, Jeff
+
+- Is UBI a red herring? What if worries about unemployment are the real underlying issue https://www.bloomberg.com/features/2017-new-unemployment/
+- Split into small groups
+- Cover questions: Is this inevitable? What is/are appropriate response(s)? What role do you have in being part of the solution? 
+
+# Turing Career Development: Automation
+
+## Session Outcomes
+
+## Mission Alignment		 
+  		  
+*Our mission is to unlock human potential by training a diverse, inclusive student body to succeed in high-fulfillment technical careers.
+  		  
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+
+### Survey Questions
+
+Survey Questions to share with instructional staff by Wednesday of the same week.
+
+1.   What value would you assign to this session (from 1-5)?
+2.   What are you taking away from this session?
+3.   What role, if any, does Turing play in taking responsibility for this issue?
+
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
+
+*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
+*   "The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"
+https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review (_The Guardian_: October 1, 2015)
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+*   ["This Is the New Face of American Unemployment"](https://www.bloomberg.com/features/2017-new-unemployment/) (Bloomberg: 2017)
```

Comment:
Love the addition of this article, but do you think students will have time for all of this pre-work? Who will send out the pre-reading? When will you send it out?